packages/cli/src/commandsConfig.ts

```diff
@@ -293,6 +293,10 @@ const commandsConfig = {
         description: '(Experimental) Format in which to write the actions script (Options: json, ethers)',
         defaultValue: 'ethers',
       },
+      {
+        flags: '-sr --skip-reference',
```

Comment:
the naming of this option does little to explain what it does (`reference` is an extremely broad concept)

perhaps a name such as `--ignore-pkgref-check` would ? there isnt a lot of pressure for it to be short because we dont want people to use it unless they know what they are doing also :sweat_smile: